.gitignore

```diff
@@ -50,3 +50,6 @@ jenkins_*.changes
 push-build.sh
 war/node_modules/
 .java-version
+
+# cf. Jenkinsfile
+/.repository/
```

Comment:
@recena Explained in the line above, specifically https://github.com/jenkinsci/jenkins/blob/master/Jenkinsfile#L44...L47